apps/api/app.py

```
@@ -6,10 +6,10 @@
 import os
 from collections.abc import AsyncGenerator
 from contextlib import asynccontextmanager
-from typing import Any
+from typing import Any, Callable, cast
```

Ruff (0.14.1)

Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)